src/audio/pipeline.c

```diff
@@ -260,7 +260,6 @@ static void pipeline_comp_period_frames(struct comp_dev *current,
 	else
```

Comment:
@IulianOlaru249 this this mostly looking good. We need some changes for the commit message because it is confusing. 

So, basically here you are doing the same kind of change into multiple files and then you are trying to catch that in the commit message I think a better commit message would be:

```
audio: Remove excess blank lines

This makes code easier to read and also silences checkpatch.pl script.
```

Remember that the commit message MUST explain *why* the commit is necessary. In your case here it is because we want to make code easier to read.

Reply:
I think the commit message has been changed now.

Reply:
@IulianOlaru249 yes the commit message looks good now! Thanks. See my other comment.

Reply:
Change the user name in .gitconfig file and changed the signed-off-by message.